examples/preprocessing/plot_ica_comparison.py

```diff
@@ -0,0 +1,69 @@
+"""
+===========================================
+Compare the different ICA algorithms in MNE
+===========================================
+
+Different ICA algorithms are fit to raw MEG data, and the corresponding maps
+are displayed.
+"""
+
+# Authors: Pierre Ablin <pierreablin@gmail.com>
+#
+#
+# License: BSD (3-clause)
+from time import time
+
+import mne
+from mne.preprocessing import ICA
+from mne.datasets import sample
+
+
+print(__doc__)
+
+###############################################################################
+# Read and preprocess the data. Preprocessing consists of:
+#
+# - EEG channel selection
+#
+# - 1-30 Hz band-pass filter
+
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=False,
```

Comment:
Shorten to `picks = mne.pick_types(raw.info)` (you are using the defaults anyway).